controllers/api/userRoutes.js

```diff
@@ -1,61 +1,222 @@
 const router = require('express').Router();
-const { User } = require('../../models');
+const { User, Event, Task } = require('../../models');
+const withAuth = require('../../utils/auth');
 
-router.post('/', async (req, res) => {
-  try {
-    const userData = await User.create(req.body);
+// GET /api/users
+router.get('/', (req, res) => {
```

Comment:
I noticed that in the mini-project, all of the get routes are in the homeRoutes.js files and the api routes just have the post and delete routes.  I am not sure if this is a convention or why they did it that way.  Not sure if this needs a change.  (I have asked Ross and Jonathan in Slack).

That all being said, this particular route is not in homeRoutes but if it was there is would be something like '/users'

Reply:
Please advise further on this issue.